omod/src/main/java/org/openmrs/module/nigeriaemr/ndrfactory/PharmacyDictionary.java

```diff
@@ -275,7 +276,65 @@ public void loadDictionary() {
         regimenCodeDescTextMap.put(80945, "Nystatin");//Added By Nelson
         regimenCodeDescTextMap.put(161364, "TDF/3TC"); //Missing NDR Code lamivudine/fenofovir from APINS Instance
         regimenCodeDescTextMap.put(165631, "Dolutegravir");// Missing NDR Code from IHVN Instance
-        regimenCodeDescTextMap.put(1674, "RIFAMPICIN/ISONIAZID/PYRAZINAMIDE/ETHAMBUTOL PROPHYLAXIS");// Missing NDR Code from IHVN Instance
+       // regimenCodeDescTextMap.put(1674, "RIFAMPICIN/ISONIAZID/PYRAZINAMIDE/ETHAMBUTOL PROPHYLAXIS");// Missing NDR Code from IHVN Instance
+
+        //Added by APIN Team 02-09-2020
+        regimenCodeDescTextMap.put(656, "Isoniazid");
+        regimenCodeDescTextMap.put(767, "Rifampicin");
+        regimenCodeDescTextMap.put(5829, "Pyrazinamide");
+        regimenCodeDescTextMap.put(745, "Ethambutol");
+        regimenCodeDescTextMap.put(438, "Streptomycin");
+        regimenCodeDescTextMap.put(1108, "Isoniazid-Ethambutol");
+        regimenCodeDescTextMap.put(1194, "Isoniazid-Rifampicin");
+        regimenCodeDescTextMap.put(1677, "Isoniazid-Rifampicin-Ethambutol");
+        regimenCodeDescTextMap.put(1131, "Isoniazid-Rifampicin-Pyrazinamide-Ethambutol");
+        regimenCodeDescTextMap.put(1674, "Isoniazid-Rifampicin-Pyrazinamide-Ethambutol-Streptomycin");
+        regimenCodeDescTextMap.put(82912,"Pyridoxine");
+        regimenCodeDescTextMap.put(160020,"Viomycin");
+        regimenCodeDescTextMap.put(84836,"Terizidone");
+        regimenCodeDescTextMap.put(73581,"Clofazimine");
+        regimenCodeDescTextMap.put(73498,"Clarithromycin");
+        regimenCodeDescTextMap.put(450,"Amoxicillin and clavulanic acid");
+        regimenCodeDescTextMap.put(160019,"Thioacetazone");
+        regimenCodeDescTextMap.put(82772,"Prothionamide");
+        regimenCodeDescTextMap.put(81457,"P-aminosalicylic acid");
+        regimenCodeDescTextMap.put(76835,"Gatifloxacin");
+        regimenCodeDescTextMap.put(80133,"Moxifloxacin");
+        regimenCodeDescTextMap.put(78788,"Levofloxacin");
+        regimenCodeDescTextMap.put(81022,"Ofloxacin");
+        regimenCodeDescTextMap.put(73449,"Ciprofloxacin");
+        regimenCodeDescTextMap.put(71060,"Amikacin");
+        regimenCodeDescTextMap.put(78385,"Kanamycin");
+        regimenCodeDescTextMap.put(72794,"Capreomycin");
+        regimenCodeDescTextMap.put(74123,"Cycloserine");
+        regimenCodeDescTextMap.put(84360,"STREPTOMYCIN");
+        regimenCodeDescTextMap.put(82900,"PYRAZINAMIDE");
+        regimenCodeDescTextMap.put(75976,"Ethionamide");
+        regimenCodeDescTextMap.put(80133,"Moxifloxacin");
+        regimenCodeDescTextMap.put(78788,"Levofloxacin");
+        regimenCodeDescTextMap.put(72794,"CAPREOMYCIN");
```

Comment:
@wonderfulguru1 there are duplicates in this map, is that intentional?

e.g 
 regimenCodeDescTextMap.put(72794,"Capreomycin");
and 
regimenCodeDescTextMap.put(72794,"CAPREOMYCIN");

Reply:
regimenCodeDescTextMap.put(71060,"AMIKACIN"); and  regimenCodeDescTextMap.put(71060,"Amikacin");
 regimenCodeDescTextMap.put(80133,"Moxifloxacin"); and regimenCodeDescTextMap.put(80133,"Moxifloxacin");

Reply:
@wandechris  we will check this revert thanks